app/services/document_service.py

```
@@ -0,0 +1,38 @@
+            # 기존 DB 초기화 (선택적)
+            clear_project_documents(project_id)
+
+            # Chroma DB 연결
+            db = Chroma(persist_directory=persist_dir, embedding_function=self.embedding_fn)
+
+            # 실제 PDF 업로드 로직 (문서 분할, 임베딩 등)
+            # 여기서는 예시로 문서 하나 추가
+            db.add_documents([object_key], chunk_size=chunk_size)  # 필요에 따라 PDF를 로컬로 다운로드 후 경로 전달
+
+            print("[INFO] 문서 업로드 성공")
+            return True
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)